src/js/components/action.jsx

```diff
@@ -62,6 +62,11 @@ export class ActionComponent extends Component {
     render() {
         const publicKeys = store.getState().app.publicKeys;
 
+        let style = {};
+        if (this.props.accentColor) {
+            style.backgroundColor = style.borderColor = `#${this.props.accentColor}`;
```

Comment:
Same for the bubble I guess? In that case it might be clashing and unreadable in some case... we might want to detect if it's a dark or light color and choose the text color based on that.